translator/isValid.go

```diff
@@ -11,7 +11,7 @@ import (
 //ErrorMessages will provide detail error messages to user
 var ErrorMessages = []string{}
 var InfoMessages = []string{}
-var ValidRetentionInDays = []string{"-1", "1", "3", "5", "7", "14", "30", "60", "90", "120", "150", "180", "365", "400", "545", "731", "1827", "3653"}
+var ValidRetentionInDays = []string{"-1", "1", "3", "5", "7", "14", "30", "60", "90", "120", "150", "180", "365", "400", "545", "731", "1827", "2192", "2557", "2922", "3288", "3653"}
```

Comment:
If it's good, please add a comment reference to this public doc https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#CWL-PutRetentionPolicy-request-retentionInDays

Reply:
Thanks, added a comment referencing this. 

I think thats a good point and we shouldn't have to manually test it but instead we can extend integration tests to verify that each retention value is still valid